### PR TITLE
Schedule and sponsor

### PR DESCRIPTION
**Pull request by Shiven110504**
Will fix the Merge Request in sometime.
Fixed the FAQ lines, sponsor email and removed sponsor block at the EOP to make page more concise

### CODE REVIEW

**Comment by Shiven110504** [comment]
Resolved issue #178 #209

**Comment by ntdkhiem** [comment]
This is very cluttering.
image
Could you add margin so that it would look like more consistent to the rest of the sections?
image

**Comment by ntdkhiem** [comment]
Could we add back **FAQ** (or something else like **Have a Question?**) at the top of this section so it will be consistent with the rest?
image

**Review by ntdkhiem — CHANGES_REQUESTED** [review]
Follow the suggestions above.

**Review by ammuiyer — APPROVED** [review]
LGTM!

File: index.html [hunk 1]
```diff
@@ -797,7 +800,7 @@ <h5 class="card-title">
           <header class="section-header" style="margin-bottom: 35px">
             <div>
               <h2 class="primary-color">Our Sponsors</h2>
-              <p class="lead">
+              <p class="lead"></p>
```
Comment (ntdkhiem) [inline index.html:803]: this closing tag here is unnecessary. Could you remove it so I can approve this PR?

**Review by ntdkhiem — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Shiven110504) [reply index.html]: Made the changes

**Review by ntdkhiem — APPROVED** [review]
LGTM

**Pull request merged**